maintenance/changeDBCluster.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+$IP = getenv( 'MW_INSTALL_PATH' );
+if ( $IP === false ) {
+	$IP = __DIR__ . '/../../..';
+}
+require_once "$IP/maintenance/Maintenance.php";
+
+class ChangeDBCluster extends Maintenance {
+	private $dbw = null;
+
+	public function __construct() {
+		parent::__construct();
+		$this->addOption( 'file', 'Path to file where the wikinames are store. Must be one wikidb name per line. (Optional, fallsback to current dbname)', false, true );
+		$this->addOption( 'db-cluster', 'Sets the wikis requested to a different db cluster.', true, true );
+	}
+
+	public function execute() {
+		global $wgCreateWikiDatabase, $wgCreateWikiDatabaseClusters, $wgDBname;
+		
+		if ( !$wgCreateWikiDatabaseClusters ) {
+			$this->fatalError( "You have not set \$wgCreateWikiDatabaseClusters." );
```

Comment:
Is this strictly a requirement? I don't see it being used elsewhere in the script.

Reply:
Also if you used '' instead of "", you'd not need to escape

Reply:
Well, no not strictly required i guess.

I'll remove, thanks!